### PR TITLE
Clarify expected output in all cases

### PR DESCRIPTION
**Pull request by damon314159**
### Description

This PR adds a small clarification to the expected outcome from one of the exercises without giving away the solution.

### Motivation

The exercise eludes to use of the modulo operator to test for divisibility, but the linked guide does not clarify that zero is a positive result for this test and non zero is a negative result. Newcomers, while understanding division, may not understand how this relates to the modulo operator without a prompt as to expected results. Informing them that they should store 0 for even and 1 for odd, combined with the link to the page containing info that includes modulo, should be sufficient for the non mathematically inclined to piece together the method for themselves, without being too easy, or telling them which operator is required.

### CODE REVIEW

**Comment by damon314159** [comment]
I'm not sure if I did this right - can someone let me know if there was something else I was supposed to do before someone will review it?

File: files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Try updating the live code below to recreate the finished example by following t
 2. Add the first two variables together and store the result in another variable.
 3. Subtract the fourth variable from the third and store the result in another variable.
 4. Multiply the results from steps **2** and **3** and store the result in a variable called `finalResult`.
-5. Check if `finalResult` is an even number using an [arithmetic operator](/en-US/docs/Learn/JavaScript/First_steps/Math#arithmetic_operators) and store the result in a variable called `evenOddResult`.
+5. Check if `finalResult` is an even number using one of the [arithmetic operators](/en-US/docs/Learn/JavaScript/First_steps/Math#arithmetic_operators). Store the result (0 for even, 1 for odd) in a variable called `evenOddResult`.
```
Comment (dipikabh) [inline files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md:28]:
```suggestion
5. Check if `finalResult` is an even number using one of the [arithmetic operators](/en-US/docs/Learn/JavaScript/First_steps/Math#arithmetic_operators). Store the result (`0` for even, `1` for odd) in a variable called `evenOddResult`.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]

**Review by dipikabh — APPROVED** [review]
This is a good clarification to add. Thank you @damon314159 👍

**Comment by dipikabh** [comment]
Congratulations on landing your first PR here!
Welcome to the MDN project 🎉

**Pull request merged**